README.rst

```diff
@@ -0,0 +1,263 @@
+h1-cli
+======
+
+h1-cli jest konsolowym narzędziem przeznaczonym do zarządzania infrastrukturą chmury HyperOne. Możesz go wykorzystywać zarówno do własnych prac administracyjnych, jak również podczas tworzenia  skryptów automatyzujących.
+
+Przegląd funkcjonalności
+------------------------
+
+* kontrola wszystkich zasobów udostępnianych przez panel administracyjny, w szczególności:
+
+  * zmiana konfiguracji serwerów - ich tworzenie, wyłączanie i kasowanie,
```

Comment:
Przyjęto terminologie spójną z webpanelem, zatem usługą są serwery, a nie np. "maszyny wirtualne" lub "instancje".

Reply:
Terminologia czeka na technical writerów na ujednolicenie.